Web Development/English_and_Numbers_Game/README.md

```diff
@@ -0,0 +1,32 @@
+# English and Numbers Game
+
+A simple and interesting game to teach kids the English alphabets and numbers 0-9.
+
+## Tech Stack :-
+
+- HTML
+- CSS
+- Javascript
+
+## Run Local
+
+* Clone this repository
+
+```bash
+git clone https://github.com/abhijeet007rocks8/Dev-Scripts
+```
+
+* Change Directory
+
+```bash
+cd Dev-Scripts/"Web Development/English_and_Numbers_Game/"
```

Comment:
why these "" comma any thing different.

Reply:
Oh, my mistake hence rectified and corrected.